tests/test_plotting.py

```diff
@@ -895,8 +896,126 @@ def test_plot_coefficients():
     return fig
 
 
+# TODO: change these plot tests as if they get split up, the regcoil stuff
```

Comment:
still todo?

Reply:
yes, I have not looked into how to basically test multiple figures from one function

Reply:
possibly this way but not sure https://stackoverflow.com/questions/50016862/grouping-tests-in-pytest-classes-vs-plain-functions